### ISSUE TITLE
Which pytorch implementation is wrong?

### ISSUE THREAD

**Issue by gdshen** [Primus-zhao/capsule#1]
> Capsule is a magic model, before implement it, i saw lots of pytorch implementation, but lots of them suffer from different mistakes, such as wrong softmax dim, wrong squash dim.

I have read this sentence in readme, do you mean the implementation https://github.com/timomernick/pytorch-capsule ?
